kafka/signed_blinded_token_issuer_handler.go

```diff
@@ -120,9 +121,15 @@ OUTER:
 			var numT = len(request.Blinded_tokens) / (issuer.Buffer + issuer.Overlap)
 			// sign tokens with all the keys in buffer+overlap
 			for i := issuer.Buffer + issuer.Overlap; i > 0; i-- {
-				var signingKey *crypto.SigningKey
+				var (
+					signingKey *crypto.SigningKey
+					validFrom  string
+					validTo    string
+				)
 				if len(issuer.Keys) > i {
 					signingKey = issuer.Keys[len(issuer.Keys)-i].SigningKey
+					validFrom = issuer.Keys[len(issuer.Keys)-i].ValidFrom.Format(time.RFC3339)
```

Comment:
Should these be - 
.StartAt
.EndAt

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/160/commits/92bd18664526af580fb74032705fbe2649b27782